modules/movie.py

```diff
@@ -22,11 +23,22 @@ def movie(jenni, input):
     if not input.group(2):
         return
     word = input.group(2).rstrip()
-    word = word.replace(" ", "+")
-    uri = "http://www.imdbapi.com/?t=" + word
+    matchObj = re.match( r'([\w\s]*)\s?,\s?(\d{4})', word, re.M|re.I)
```

Comment:
I don't know anything about this API, but are we always expecting this exact regex? It seems like ``` , 0000``` would pass, which is unexpected...